cpp/cmake/thirdparty/get_cuvs.cmake

```diff
@@ -48,8 +48,8 @@ function(find_and_configure_cuvs)
       BUILD_EXPORT_SET    cuml-exports
       INSTALL_EXPORT_SET  cuml-exports
       CPM_ARGS
-        GIT_REPOSITORY         https://github.com/${PKG_FORK}/cuvs.git
-        GIT_TAG                ${PKG_PINNED_TAG}
+        GIT_REPOSITORY         https://github.com/jinsolp/cuvs.git
```

Comment:
Needs to be reverted once [this PR](https://github.com/rapidsai/cuvs/pull/785) in cuvs is merged